### PR TITLE
Analytics change for v2.0 except VCACA images

### PR DESCRIPTION
**Pull request by dpatel257**

### CODE REVIEW

**Comment by jhou5** [comment]
dldt_gst_gvapython test failed as known issue.

File: XeonE3/centos-7/analytics/gst/Dockerfile.m4 [hunk 1]
```diff
@@ -1,9 +1,7 @@
-
-FROM centos:centos7 AS build
-WORKDIR /home
-define(`BUILD_LINKAGE',shared)dnl
-
-include(build-tools.m4)
+include(begin.m4)
+include(centos-repo.m4)
+include(cmake.m4)
+include(git.m4)
```
Comment (jhou5) [inline XeonE3/centos-7/analytics/gst/Dockerfile.m4:4]: What issue to be fixed by including git.m4

**Review by jhou5 — COMMENTED** [review]
CentOS 7 test failed.
Step 58/147 : RUN cd /opt/build &&     git clone ${SVT_VP9_REPO} -b v0.2.1 --depth 1 &&     cd SVT-VP9/Build/linux &&     (. /opt/rh/devtoolset-9/enable && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_INSTALL_LIBDIR=/usr/local/lib64 -DCMAKE_ASM_NASM_COMPILER=yasm ../.. &&     make -j $(nproc) ) &&     make install DESTDIR=/opt/dist &&     make install
 ---> Running in 14ed20579659
Cloning into 'SVT-VP9'...
Note: checking out '11d000baab0f7f02085cab9fbd369e6c14d3926c'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

/bin/sh: /opt/rh/devtoolset-9/enable: No such file or directory
The command '/bin/sh -c cd /opt/build &&     git clone ${SVT_VP9_REPO} -b v0.2.1 --depth 1 &&     cd SVT-VP9/Build/linux &&     (. /opt/rh/devtoolset-9/enable && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_INSTALL_LIBDIR=/usr/local/lib64 -DCMAKE_ASM_NASM_COMPILER=yasm ../.. &&     make -j $(nproc) ) &&     make install DESTDIR=/opt/dist &&     make install' returned a non-zero code: 1
Xeon/centos-7/analytics/gst/CMakeFiles/build_xeon_analytics_gst_centos7.dir/build.make:57: recipe for target 'Xeon/centos-7/analytics/gst/CMakeFiles/build_xeon_analytics_gst_centos7' failed
make[2]: *** [Xeon/centos-7/analytics/gst/CMakeFiles/build_xeon_analytics_gst_centos7] Error 1
CMakeFiles/Makefile2:9176: recipe for target 'Xeon/centos-7/analytics/gst/CMakeFiles/build_xeon_analytics_gst_centos7.dir/all' failed
make[1]: *** [Xeon/centos-7/analytics/gst/CMakeFiles/build_xeon_analytics_gst_centos7.dir/all] Error 2
Makefile:94: recipe for target 'all' failed
make: *** [all] Error 2

File: XeonE3/centos-7/analytics/gst/Dockerfile.m4 (hunk 1 above)
Comment (dpatel257) [reply XeonE3/centos-7/analytics/gst/Dockerfile.m4:4]: New 'meson' based builds in GST world, some of them calls to 'git -C'. '-C' options was later introduced in git. And git that comes through yum in C7, is older and would not support that '-C', to make this work, we need to build newer git from source.

File: Xeon/centos-7/analytics/ffmpeg/Dockerfile.m4 [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
 include(svt-av1.m4)
-#include(svt-vp9.m4)
-#include(libjsonc.m4)
-#include(librdkafka.m4)
-include(opencv.m4)
+include(svt-vp9.m4)
 include(dldt-ie.m4)
+include(libjsonc.m4)
+include(librdkafka.m4)
+include(media-driver.m4)
```
[the remark this answers is not in the document]
Comment (dpatel257) [reply Xeon/centos-7/analytics/ffmpeg/Dockerfile.m4:20]: I guess that is mistake, let me double check all 'Xeon' images. Thanks for catching that

File: XeonE3/centos-7/analytics/gst/Dockerfile.m4 (hunk 1 above)
Comment (xwu2git) [reply XeonE3/centos-7/analytics/gst/Dockerfile.m4:4]: I don't see this issue during the OWT centos build. It also includes gst.

Comment (dpatel257) [reply XeonE3/centos-7/analytics/gst/Dockerfile.m4:4]: Let me check which exact 'gst' component needed it.

File: Xeon/centos-7/analytics/ffmpeg/Dockerfile.m4 (hunk 2 above)
[the remark this answers is not in the document]
Comment (dpatel257) [reply Xeon/centos-7/analytics/ffmpeg/Dockerfile.m4]: Fix the 'Media-driver' part.

About the error on 'SVT-VP9'. It came from merging the upstream/v2.0. Tested Xeon/C7/media/ffmpeg. Same error appears.

File: XeonE3/centos-7/analytics/gst/Dockerfile.m4 (hunk 1 above)
Comment (dpatel257) [reply XeonE3/centos-7/analytics/gst/Dockerfile.m4]: Removed custom git, and not facing this anymore.

**Long story**: For some reason I had included custom built meson.m4 in C7 images, which installs Meson 0.53.1 as opposed to C7 yum install meson, which installs 0.55.1. This older meson gave problem with git, hence I wrote git.m4.

 But referring to @xwu2git 's owt recipe, I realized we dont need to source built meson for C7, after removing meson.m4 from those images, I didn't see git errors. Hence, removed git.m4.

**Review by jhou5 — APPROVED** [review]

**Pull request merged**